app/src/main/java/com/better/alarm/presenter/AlarmsListActivity.kt

```diff
@@ -202,6 +208,28 @@ class AlarmsListActivity : AppCompatActivity() {
             }.apply { }
     }
 
+    private fun showMuteAlert() {
+        val audio: AudioManager = this.getSystemService(AUDIO_SERVICE) as AudioManager
+        if(audio.getStreamVolume(AudioManager.STREAM_ALARM)<=1) {
+            //implementation of snackbar
+//            Toast.makeText(this,"" + audio.getStreamVolume(AudioManager.STREAM_ALARM) , Toast.LENGTH_LONG).show();
+            val rootView = findViewById<View>(android.R.id.content)
+            val toastText = "Warning: Alarm volume is low"
+            val volumeSnack = Snackbar.make(rootView, toastText, Snackbar.LENGTH_INDEFINITE)
```

Comment:
How does this snackbar interact with another one? I think it needs a different color.